submissions/Temu4/html_css_popup/index.html

```diff
@@ -0,0 +1,169 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <link rel="stylesheet" href="style.css" />
+    <title>Booble menu</title>
+  </head>
+  <body>
+    <div class="menu">
+      <div class="menu-bar">
+        <!-- Link for mail -->
+        <a href="#" tabindex="1">Bmail</a>
+        <!-- Link for images -->
+        <a href="#" tabindex="2">Images</a>
+        <!-- Link for apps -->
+        <a class="icon-apps bar" href="#" title="Приложения Booble" tabindex="3"
+          ><img src="img/icon-apps.png" alt="Icon apps" />
+        </a>
+        <!-- Container with apps list -->
+        <div class="apps-list">
+          <table>
```

Comment:
In my opinion use for showing list in <table>  is not good practice.
Try using blocks using the display style `inline-block`
Check the alignment of the content in the center, after the appearance of the scroll, everything did not move to the left

Reply:
thanks for the advice! I'll try to fix it.

Reply:
[link](https://en.bem.info/) Good methodology for naming class.
Recomend for read =)